src/challenges.js

```diff
@@ -19,11 +27,24 @@ const counter = () => {
   // Example: const newCounter = counter();
   // newCounter(); // 1
   // newCounter(); // 2
+  console.log(count);
+  count++;
```

Comment:
While I understand what you're trying to do here I feel like it's just barely skirting by. It doesn't really provide  what the instructions requested. `Return a function that when invoked increments and returns a counter variable.`